reporters/GlobalSummaryReporter.js

```diff
@@ -699,7 +648,85 @@ class GlobalSummaryReporter {
           Object.keys(tokenHolderStats.countPeriodTokenHolders["prevPeriod"]).length
       )
     };
-    console.table(allTokenStatsTable);
+    console.table(allTokenHolderStatsTable);
+  }
+
+  async _generateExchangeStats() {
```

Comment:
Just curious -- are there subgraphs for balancer?

Reply:
Yep! Follow up PR will add that to this Reporter. Also a bunch of these event querying methods can be replaced with subgraph calls to UMA.